### DISCUSSION TITLE
Cannot install when in other countries

### DISCUSSION THREAD

**Discussion by bmthuat** [Digitaler-Impfnachweis/covpass-android#145] [Ideas]
I'm currently in Singapore with a Singapore payment profile. Play Store is not letting me install the app (This item is not available in your country)

Germany now allows vaccinated travellers from other countries and some governments already issued EU compatible vaccination certificates. Is it possible to let users from other countries install the app?

Thanks!

**Comment by jleufgen** [comment]
@bmthuat I understand your desire to have the apps in other App Stores as well. Currently, the CovPass is only available in the German and the US App Store.
Nevertheless, for now, there are no intentions of opening the app to further app stores. If you have another app store (not US or DE), we can only forward you to the Corona Warn App.
